parlai/tasks/bot_adversarial_dialog/README.md

```diff
@@ -0,0 +1,16 @@
+Task: Bot-Adversarial Dialogue Dataset
+===========================
+Description: Dialogue datasets labeled with offensiveness from Bot-Adversarial Dialogue task
+=========================== 
+Data has been released under the CC BY-NC license.
```

Comment:
which license is this? was this required by legal review?

Reply:
Should link to the license. (but it's creative commons attribute non-commercial)

Reply:
will get rid of the license.